app/models/invoice.rb

```diff
@@ -6,6 +6,8 @@ class Invoice < ApplicationRecord
   validates :amount, presence: true, numericality: { greater_than: 0 }
   validates :due_date, presence: true
 
+  monetize :amount, as: :amount_brl
```

Comment:
I think we don't need to rename the attribute here. If I'm not wrong this `as` is to rename the attribute, right?
```suggestion
  monetize :amount
```

Reply:
Sounds good. This `as` can be removed as we can see in the docs we should add `as` when we need to put the `Money` object in an alias column.

https://github.com/RubyMoney/money-rails#usage-example

Reply:
Hey guys!

I've tried your suggestion, however it would only work if the column was renamed as `amount_cents` .

Using the `as` argument is just a workaround when we we have a db column with another name. With `as`, the Invoice model will have an `amount_brl` attribute which is a Money object.

@m-pereira what do you think?

Reply:
hmm, nice catch! We can rename the column, in this case, 😄 you can add it into your migration to convert the amounts to integer

Reply:
Ok! :fist_oncoming:

Reply:
Done!